hazelcast/src/test/java/com/hazelcast/util/executor/CancellableDelegatingFutureTest.java

```diff
@@ -35,17 +34,15 @@ public void testInnerFutureThrowsCancellationExceptionWhenOuterFutureIsCancelled
             try {
                 future.getFuture().get();
                 fail();
-            } catch (ExecutionException expected) {
-                assertTrue(expected.getCause() instanceof CancellationException);
+            } catch (CancellationException expected) {
```

Comment:
Right, plus the check is localized with the invocation that should throw the exception instead of a blanket `@Test(expected = Exception.class)` (this is not the case here, but in terms of general usefulness).